src/main/java/timify/com/subject/SubjectController.java

```diff
@@ -79,4 +78,26 @@ public ApiResponse<updateTitleNameDto> updateTitle(@RequestBody @Valid subjectRe
     return ApiResponse.of(SuccessStatus.TITLE_CHANGE_SUCCESS, updateTitleNameDto);
   }
 
+  @PutMapping("/{id}/store")
+  @Operation(summary = "항목 보관함 이동 API", description = "항목을 보관함으로 이동시키는 API 입니다.")
+  public ApiResponse<Long> storeSubject(@PathVariable Long id) {
```

Comment:
```suggestion
    public ApiResponse<Long> storeSubject(@PathVariable(name = "id") Long id) {
```
그리고 pathVariable도 웬만해서는 (name="id")를 붙여서 매핑을 정확하게 지정해주는게 좋은 것 같아요..! 테스트해보니까 저는 
Name for argument of type [java.lang.Long] not specified, and parameter name information not available via reflection. Ensure that the compiler uses the '-parameters' flag.] with root cause 
이런 에러가 뜨더라구요..

Reply:
확인했습니다.